### PR TITLE
added score, made the game harder. this closes #33

### LINKED ISSUE #33: Key improvements needed for game
- [x] game has to end when obstacle and player touch
- [x] game is too easy atm

### PR DESCRIPTION
**Pull request by janamano**

### CODE REVIEW

File: ios/Podfile.lock [hunk 1]
```diff
@@ -253,6 +253,8 @@ PODS:
     - React-cxxreact (= 0.61.5)
     - React-jsi (= 0.61.5)
     - ReactCommon/jscallinvoker (= 0.61.5)
+  - RNCAsyncStorage (1.9.0):
```
Comment (aniprasad) [inline ios/Podfile.lock:256]: I don't think you need to check this file in

File: src/components/Constants.js [hunk 2]
```diff
@@ -37,28 +38,28 @@ export default Constants = {
 
   // Individual Badge Badges
   WORLD_COMPLETION_FIRE: 1,
-  WORLD_COMPLETION_FIRE_IMG: require("../assets/badges/world_completion.png"),
-  WORLD_COMPLETION_FIRE_IMG_RAW: "../assets/badges/world_completion.png",
+  WORLD_COMPLETION_FIRE_IMG: require("../assets/badges/lesson_completion.png"),
```
Comment (aniprasad) [inline src/components/Constants.js:41]: You might have to merge the latest version of the code into your branch. This seems a bit outdated

File: ios/Podfile.lock (hunk 1 above)
Comment (janamano) [reply ios/Podfile.lock:256]: this file is under version control, i thought it would be best to add
TBH, i dont know what this file is for Lol

Comment (aniprasad) [reply ios/Podfile.lock:256]: Ahh ok

File: src/components/Physics.js [hunk 3]
```diff
@@ -2,72 +2,135 @@ import Matter from 'matter-js';
...
 }
+export const collisionCheck = (otherBody, playerBody, dispatch) => {
+    let collision = Matter.SAT.collides(otherBody, playerBody);
+    if (collision.collided) {
+        dispatch({ type: "game-over" });
+    }
+}
 
-const Physics = (entities, { touches, time }) => {
+// export const addObstacleToGame = (x, world, entities) => {
```
Comment (aniprasad) [inline src/components/Physics.js:45]: Is there a reason the commented code is checked in? If its important keep it, otherwise remove it

**Pull request merged**